server/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
```

Comment:
Next time consider something like `Dexie.js` so you don't have to spend so much time on setup to get something working quickly. 

All the groundwork you did is impressive and is very useful in case you continue to build it up into a finished product.